ghc-mod.cabal

```diff
@@ -201,6 +205,8 @@ Executable ghc-mod
                       , mtl       < 2.3 && >= 2.0
                       , ghc       < 7.11
                       , fclabels  ==2.0.*
+                      , optparse-applicative ==0.11.*
+                      , ansi-wl-pprint ==0.6.*
```

Comment:
Hmm so we're depending on two pretty printing libraries now, is there any reason to use this one vs `pretty` or the other way around?